src/components/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import Filter from "../Filter";
+import Pagination from "../Pagination";
+import { Container } from "../ui/Layout/StyledLayout";
+import Text from "../ui/Text";
+import IncidentsList from "../IncidentsList";
+import { incidentPropTypes } from "../../constants/propTypes";
+import ErrorMessage from "../ui/ErrorMessage";
+import ErrorBoundary from "../ErrorBoundary";
+
+class HomePage extends Component {
+  componentDidMount = () => {
```

Comment:
why do you need to define `componentDidMount` in such way?

```suggestion
  componentDidMount() {
```

Reply:
No need to use the arrow function there,  I just use a vscode snippets package for autocomplete React/Redux code and it has the arrow function in the componentDidMount snippet